packages/cli/README.md

```diff
@@ -0,0 +1,27 @@
+# mockyeah-cli
```

Comment:
I think this folder should be called `mockyeah-cli` rather than just `cli` so it matches the package name on npm.

Reply:
I agree.